### PR TITLE
Add data::GuideRateValue mpiserializer test

### PR DESCRIPTION
**Pull request by akva2**
Have to extend the primitive serializer with std::bitset support.

Downstream of https://github.com/OPM/opm-common/pull/3121

### CODE REVIEW

File: opm/simulators/utils/ParallelRestart.cpp [hunk 1]
```diff
@@ -409,6 +409,27 @@ struct Packing<data::Segment>
     }
 };
 
+template <size_t Size>
```
Comment (bska) [inline opm/simulators/utils/ParallelRestart.cpp:412]: Is there a reason for using `size_t` instead of `std::size_t` here?

**Review by bska — COMMENTED** [review]
Apart from `size_t` vs. `std::size_t` (`<cstddef>`) this looks good to me.

File: opm/simulators/utils/ParallelRestart.cpp (hunk 1 above)
Comment (akva2) [reply opm/simulators/utils/ParallelRestart.cpp:412]: no reason, just a slip. fixed now.

**Review by bska — APPROVED** [review]
Thanks.  I'll merge into master.

**Pull request merged**